client/stylesheets/_modal.scss

```diff
@@ -212,17 +212,17 @@
 }
 
 /* Radio Buttons */
-.modOfferRequestPaymentWrap ul li {
+.radioWrap ul li {
 	clear: both;
 	color: #999 !important;
 	display: inline-block;
 }
 
-.modOfferRequestPaymentWrap ul li:nth-child(2), .modOfferRequestPaymentWrap ul li:nth-child(3) {
+.radioWrap ul li:nth-child(2), .radioWrap ul li:nth-child(3) {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `radioWrap` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line